### PR TITLE
Feature/rabbitmq integration

### PR DESCRIPTION
**Pull request by harsha-wedaa**
- Modified the flow of code generation
- Download on the prototypes tab
- Toast on completion of code generation
- polling on prototypes tab for the availability of prototype for code generation
- Implemented versioning on the ui

### CODE REVIEW

File: src/pages/Review/CodeReview.js [hunk 1]
```diff
@@ -114,17 +114,17 @@ function CodeReview({
                     body: JSON.stringify(Data),
                 });
                 blueprintId = response.headers.get('blueprintid');
-                const blob = await response.blob();
+                await new Promise(resolve => setTimeout(resolve, 10000));
                 setIsGenerating(false);
-                saveAs(blob, `${Data.projectName}.zip`);
+                // saveAs(blob, `${Data.projectName}.zip`);
```
Comment (raxkumar-wedaa) [inline src/pages/Review/CodeReview.js:119]: can remove if not required !

File: src/pages/Review/Review.js [hunk 2]
```diff
@@ -155,6 +155,38 @@ export const ReviewFlow = ({
...
+                    history.replace('/prototypes');
+                } else {
+                    history.replace('/architectures');
+                }
+            } else {
+                history.push('/canvasToCode');
+            }
+        }
+    }
+    // else{
```
Comment (raxkumar-wedaa) [inline src/pages/Review/Review.js:185]: remove this commeneted else {} block if not used

**Review by raxkumar-wedaa — APPROVED** [review]
Resolve the review comments

File: src/pages/Review/Review.js [hunk 3]
```diff
@@ -270,7 +299,20 @@ export const ReviewFlow = ({
...
+                                    onClick={handleDownload}
+                                    colorScheme="blue"
+                                    position="absolute"
+                                    top="120px"
+                                    cursor={reviewData.latestCodeGenerationStatus=="COMPLETED" ? '': 'not-allowed'}
+                                    zIndex={999}
+                                    icon={<DownloadIcon />}
+                                />
+                            </Tooltip>
+                        )} */}
```
Comment (rohith-wedaa) [inline src/pages/Review/Review.js:315]: @harsha-wedaa Please enable this Tooltip if required. Else remove the code.

**Review by rohith-wedaa — APPROVED** [review]

**Pull request merged**